core/integration/features/verified/0070-MKTD-market-decimal-places-negative.feature

```diff
@@ -0,0 +1,176 @@
+Feature: Allow markets to be specified with a smaller number of decimal places than the underlying settlement asset
```

Comment:
I don't understand why it's under `0070-MKTD-...` since we want to test negative PDPs specifically.